docs/en/interfaces/cli.md

```diff
@@ -1,63 +1,143 @@
+
+<img src={require('../_snippets/images/connection-details-native.png').default}
+  class="image"
+  alt="ClickHouse Cloud Native TCP connection details"
+  style={{width: '40em'}} />
+
+
+### Storing connections in a configuration file {#connection-credentials}
+
+You can store connection details for one or more ClickHouse servers in a [configuration file](#configuration_files).
+
+The format looks like this:
+```xml
+<config>
+    <connections_credentials>
+        <name>default</name>
+        <hostname>hostname</hostname>
+        <port>9440</port>
+        <secure>1</secure>
+        <user>default</user>
+        <password>password</password>
+    </connections_credentials>
+</config>
 ```
 
-## Usage {#cli_usage}
+See the [section on configuration files](#configuration_files) for more information.
+
+:::note
+To concentrate on the query syntax, the rest of the examples leave off the connection details (`--host`, `--port`, etc.). Remember to add them when you use the commands.
+:::
 
-The client can be used in interactive and non-interactive (batch) mode.
 
-### Gather your connection details
-<ConnectionDetails />
+## Batch mode {#batch-mode}
 
-### Interactive
+Instead of using ClickHouse Client interactively, you can run it in batch mode.
 
-To connect to your ClickHouse Cloud service or any ClickHouse server using TLS and passwords, specify port `9440` (or `--secure`) and provide your username and password:
+You can specify a single query like this:
 
 ```bash
-clickhouse-client --host <HOSTNAME> \
-                  --port 9440 \
-                  --user <USERNAME> \
-                  --password <PASSWORD>
+$ clickhouse-client "SELECT sum(number) FROM numbers(10)"
+45
 ```
 
-To connect to a self-managed ClickHouse server you will need the details for that server.  Whether or not TLS is used, port numbers, and passwords are all configurable.  Use the above example for ClickHouse Cloud as a starting point.
+You can also use the `--query` command-line option:
 
+```bash
+$ clickhouse-client --query "SELECT uniq(number) FROM numbers(10)"
+10
+```
 
-### Batch
+You can provide a query on `stdin`:
 
-To use batch mode, specify the ‘query’ parameter, or send data to ‘stdin’ (it verifies that ‘stdin’ is not a terminal), or both. Similar to the HTTP interface, when using the ‘query’ parameter and sending data to ‘stdin’, the request is a concatenation of the ‘query’ parameter, a line feed, and the data in ‘stdin’. This is convenient for large INSERT queries.
+```bash
+$ echo "SELECT avg(number) FROM numbers(10)" | clickhouse-client
+4.5
+```
+
+Inserting data:
+
+```bash
+$ echo "Hello\nGoodbye" | clickhouse-client --query "INSERT INTO messages FORMAT CSV"
+```
 
-Examples of using the client to insert data:
+When `--query` is specified, any input is appended to the request after a line feed.
 
-#### Inserting a CSV file into a remote ClickHouse service
+**Inserting a CSV file into a remote ClickHouse service**
```

Comment:
why bold and not sub headings? this will impact SEO

Reply:
They h4, likely wont impact headers too much. If you dont feel like we need the menu use bold